src/main/io/vtx_smartaudio.c

```diff
@@ -615,6 +655,42 @@ void saSetPowerByIndex(uint8_t index)
     saQueueCmd(buf, 6);
 }
 
+void saSetPowerByIndex(uint8_t index)
+{
+    saDevSetPowerByIndex(index);
+    vtxSettingsSavePowerByIndex(index + 1);
+}
+
+static bool saEnterInitBandChanAndPower(uint8_t band, uint8_t channel, uint8_t power)
+{
+    if (!saValidateBandAndChannel(band, channel))
+        return false;
+    saDevSetBandAndChannel(band-1, channel-1);
```

Comment:
prefer whitespace (` `) both sides of operators.  e.g. `x + 1` not `x+1`.  There are a few places in this PR that need updating.